src/Symfony/Component/Config/Definition/Exception/TreeWithoutRootNodeException.php

```diff
@@ -1,19 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Component\Config\Definition\Exception;
-
-/**
- * @author Roland Franssen <franssen.roland@gmail.com>
- */
-class TreeWithoutRootNodeException extends \RuntimeException
```

Comment:
maybe we forgot to mark this `@internal` in 4.x ... :sweat_smile:

Reply:
let's add it #31944 :)